app-office/moneydance/files/moneydance-bin

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Find JAVA
+JAVA_OPTS="-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx2048m"
+MONEYDANCE_LIBS="/usr/share/moneydance/lib"
+OPENJDK11="/usr/lib/jvm/openjdk-11"
```

Comment:
That path is still wrong. I've just spent 2 hours building openjdk just to confirm that.

Reply:
Okay, because this was a symlink, right? Changed.

Reply:
No, because it doesn't exist at all in normal installation. I guess you unmasked the masked flag.

Reply:
Oh, good to know, again some black box java magic ;-) Should now fit, since the new path does exist here too.